src/main/java/com/ubc/cpsc304/hotelify/entity/Review.java

```diff
@@ -0,0 +1,27 @@
+package com.ubc.cpsc304.hotelify.entity;
+
+import javax.persistence.*;
+
+import lombok.Data;
+
+@Data
+@Entity
+public class Review {
+
+    @Id
+    @Column(nullable = false, updatable = false, unique = true)
+    private Long review_id;
+
+    @Column(nullable = false, columnDefinition = "VARCHAR(255) DEFAULT ''")
+    private String username;
```

Comment:
this is a foreign key to our customer table right? you need to specify the relationship and use `@ManyToOne`